hadoop-ozone/common/src/main/java/org/apache/hadoop/ozone/om/helpers/OmBucketArgs.java

```diff
@@ -343,9 +357,10 @@ public static OmBucketArgs getFromProtobuf(BucketArgs bucketArgs) {
                 bucketArgs.getOwnerName() : null);
     // OmBucketArgs ctor already has more arguments, so setting the default
     // replication config separately.
-    omBucketArgs.setDefaultReplicationConfig(
-        new DefaultReplicationConfig(bucketArgs.getDefaultReplicationConfig()));
-
+    if (bucketArgs.hasDefaultReplicationConfig()) {
```

Comment:
I think you can fix this issue with just this `if` statement added here, but I am not 100% certain. It kind of depends if we need a way to remove a default rep config from a bucket.

Reply:
Updated the patch.